.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,40 @@
+name: V9 Prerelease Builds
+
+on:
+  push:
+    branches:
+      - v9
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  canary_release:
+    name: V9 Prerelase
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
+
+      - name: Setup Node.js 14.x
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
+
+      - name: Install Dependencies
+        run: yarn
+
```

Comment:
If this is our release pipeline couldn't we add the test command here?

Reply:
I would recommend integrating it into the `yarn release` command, eg. changing

```
"release": "yarn build && yarn changeset publish"
```

to

```
"release": "yarn test && yarn build && yarn changeset publish"
```

Reply:
(For context, this is a recommendation from the Changesets project. The reasoning is that you can run `yarn release` locally, and it'd be good to have it run tests there, instead of only doing it in the GH Actions pipeline.)